readme.md

```diff
@@ -0,0 +1,67 @@
+# cvsclone
+
+Utility to clone cvs repositories over the cvspserver interface.
+
+As ```git cvsimport``` is pretty slow over the wire (and as [cvs2git](http://cvs2svn.tigris.org/cvs2git.html) can do a substantially better job if there are a lot of branches), it is often better to "clone" a whole CVS repository. Enter cvsclone!
+
+## Features
+
+- Works with anonymous access. Will read $HOME/.cvspass if needed
+- Can clone corrupt repositories: writes ,v files directly, does not need rcs (For example, ccvs module has archives that go backwards in time).
+
+## Building
+
+You will need [Flex](http://flex.sourceforge.net/) installed to create cvsclone. You can normally find this in your package manager i.e:
+
+```bash
+> apt-get install flex
+```
+
+Use [Homebrew](http://brew.sh/) on OS X:
+```bash
+> brew install flex
+```
+
+Once installed, clone this repository and build using **make**:
+
+```bash
+> git clone
+> cd cvsclone
+> make
+```
+
+The **cvsclone** binary should be output into the same folder and ready to use.
+
+## Using
+
+Set up your $CVSROOT to point to your repository of choice. You will need to log into CVS first so that your credentials are on the machine:
+
+```bash
+> set $CVSROOT=:pserver:your.user@mycvsserver.com:/SomeRepo
+> cvs login
+```
+
+To clone the repository, use ```./cvsclone -d <connect pserver string> <module>```. For example:
+
+```bash
+> ./cvsclone -d $CVSROOT mymodule
+```
+
+Your repository should be converted and placed in a folder with the modules name.
+
+## History
+
+Originally written by Peter Backes. Check the header in [cvsclone.i](./cvsclone.i) for more details.
+
+Additional fixes by [Mateusz Czapliński](https://github.com/akavel/cvsclone) (which this version is based on) allowing to pipe the output of "cvs rlog" into a file, editing the file to fix errors (such as geniuses who put the cvs log of moved files into a commit message) and use that file as input instead.

```

Comment:
Please fix the start of the sentence to below instead: 

> Additional fixes by [Johannes Schindelin](http://repo.or.cz/w/cvsclone.git)... 

Also, you're more than welcome to add info about your contributions too!